lib/sync/pull-request.js

```diff
@@ -9,12 +12,33 @@ const { getPullRequests: getPullRequestQuery } = require('./queries');
  * @returns {any} TBD
  */
 exports.getPullRequests = async (github, repository, cursor, perPage) => {
-  const { edges } = (await github.graphql(getPullRequestQuery, {
+  let edges;
+  const vars = {
     owner: repository.owner.login,
     repo: repository.name,
     per_page: perPage,
     cursor,
-  })).repository.pullRequests;
+  };
+
+  let asyncTags = [];
+  await statsd.asyncTimer(
+    async () => {
+      try {
+        ({ edges } = (await github.graphql(getPullRequestQuery, vars)).repository.pullRequests);
+        asyncTags.push('status:200');
+      } catch (error) {
+        if (error instanceof RequestError) {
```

Comment:
We may be able to use duck-typing here. But this is 100% ignorable:

```suggestion
        if (error.status) {
```

This _may_ be able to drop the `RequestError` dependency and import? I dunno if that's coming in for a different reason though!